src/Compilers/VisualBasic/Test/Semantic/Semantics/Conversions.vb

```diff
@@ -605,7 +605,8 @@ End Class
                     New TypeAndValue(uint64Type, CULng(18)),
                     New TypeAndValue(decimalType, CDec(-11.3)),
                     New TypeAndValue(doubleType, CDbl(&HF000000000000000UL)),
-                    New TypeAndValue(doubleType, CDbl(&H70000000000000F0L)),
```

Comment:
>&H70000000000000F0L [](http://example.com/codeflow?start=54&length=19)

Would it make sense to use the other value from the description, `&H7000000000000400L`, instead of this case?